src/derivatives/imaging.md

```diff
@@ -14,20 +14,19 @@ Template:
             <source_entities>[_space-<space>][_res-<label>][_den-<label>][_desc-<label>]_<suffix>.<extension>
 ```
 
-Volumetric preprocessing does not modify the number of dimensions, and so
-the specifications in [Preprocessed or cleaned data][common_preproc]
-apply.
-The use of surface meshes and volumetric measures sampled to those meshes is
+Volumetric preprocessing does not modify the number of dimensions, and so the
+specifications in [Preprocessed or cleaned data][common_preproc] apply. The use
+of surface meshes and volumetric measures sampled to those meshes is
 sufficiently similar in practice to treat them equivalently.
 
-When two or more instances of a given derivative are provided with resolution
-or surface sampling density being the only difference between them, then the
-[`res`](../appendices/entities.md#res) (for *resolution* of regularly sampled N-D data) and/or
-[`den`](../appendices/entities.md#den) (for *density* of non-parametric surfaces)
-entities SHOULD be used to avoid name conflicts.
-Note that only files combining both regularly sampled (for example, gridded)
-and surface sampled data (and their downstream derivatives) are allowed
-to present both [`res`](../appendices/entities.md#res) and
+When two or more instances of a given derivative are provided with resolution or
+surface sampling density being the only difference between them, then the
+[`res`](../appendices/entities.md#res) (for _resolution_ of regularly sampled
```

Comment:
I take it you just ran remark? It doesn't like its own output. I hope you still have the pre-reformat changes in your buffer.